monkey/infection_monkey/payload/payload.py

```diff
@@ -0,0 +1,21 @@
+from abc import abstractmethod
+
+import infection_monkey.payload
+from infection_monkey.utils.plugins.plugin import Plugin
```

Comment:
Just import plugin in `__init__.py`. The import would then become `from infection_monkey.utils.plugins import Plugin` which is shorter and more readable